package.json

```diff
@@ -1,14 +1,15 @@
 {
-  "name": "@jenny-s51/patternfly-component-schemas",
-  "version": "1.0.1",
+  "name": "@patternfly/patternfly-component-schemas",
+  "version": "1.1.0",
```

Comment:
You'll probably want to have this as `1.0.0`, semantic-release actually doesn't use or update the version here. But again not a blocker.